_posts/2018-11-29-async-update.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: "Async in Rust, circa 2018"
+date: 2018-11-29
+author: Aaron Turon
+---
+
+The Rust 2018 Edition, and the end of 2018, are fast approaching! While we
+didn't manage to ship async/await as part of the edition itself, the
+community has made quite a lot of progress toward that goal. This post
+summarizes the state of play, and announces the publication several crates
+intended to facilitate use of async/await on the nightly ecosystem.
+
+# Why async/await
+
+Before delving into the current status, it's worth taking a moment to recap
+the core motivations for async/await, and its special importance for Rust.
+
+Async/await notation is a way of making asynchronous programming more
```

Comment:
nit of nits: I haven't been capitalizing `async` in `async/await` because it's a keyword, like `for` loops